packages/node-integration-tests/package.json

```diff
@@ -16,10 +16,8 @@
     "@segment/analytics-node": "workspace:^",
     "@types/analytics-node": "^3.1.9",
     "@types/autocannon": "^7",
-    "@types/express": "^4",
```

Comment:
This is a dev dependency that is already used by other packages, so we might as well just share it.